working up an email to call for adoption (#4)
datatrails/draft-bryce-cose-merkle-mountain-range-proofs: addoption-call.md

If the old proof data is not a strict prefix of the new proof data, the log has been inconsistently updated and the element has moved.

There are many ways this can be shown for various VDS's. With MMR's it is particularly convenient and requires no actions specialised to that goal.

6. Elements can be verified without revealing the item of interest.

When bounded replication is simple to accomplish, the verfier can maintain a local window on the log that makes it un-necessary to contact it for every verification.
Because the verifiable log is highly replicable, elements can be verified without revealing the item of interest to the log operator.
The linear, addition ordered, nature of the log means a verifier can simply maintain a local "window" on the replicated log, discarding sections that are no longer needed.

For certain, especialy as this is an "authors first draft", there is a journey to be made for this draft in in form, contenent and syntax.
I'm very happy to take direction and contributions to that effect.

But in the mean time, do people feel there enough here to justify the time and attention that would be needed to take it forward ?

Best Regards,

Robin Bryce

> 

Author: note to self: don't merge to main

Independent of blockchains, the draft leans heavily on work by Leonid Reyzin and Sophia Yakoubov, particularly their paper on "Efficient Asynchronous Accumulators for Distributed PKI" -- https://eprint.iacr.org/2015/718.pdf.

Their work provides a formal model that applies to the emergent benefits of MMRs, which is particularly important for the formulation of proofs of inclusion and consistency specified in the draft.

I should note that while I’ve made Leonid and Sophia aware of this draft, and they confirmed that the basic link between MMRs and their work makes sense, they have not reviewed the draft or otherwise endorsed it.

On a more personal note, this draft is the one I wish had been available when I first became convinced that MMRs were a great fit for more application-centric, "eye of the beholder-valued" transparency use cases.
It is also the result of poring over various implementations and considering which, if any, differences mattered.

Here are 6 properties that I think particularly help lower the barrier to ensuring trustworthy transparency systems:

1. Immediate replication after element addition is safe; the log data does not change after addition.

> While regular checkpoints and consistency proofs can be requested for any Merkle-based log, being able to instantly replicate the VDS data puts the verifier in control of the frequency and granularity of verification.
> Since each replicated (and attested) log is as reliable as the original, this can also significantly improve scalability.

Reviewer: I believe you're talking about the MMR comparison, which wasn't as obvious. Suggest you include "MMR", to remove ambiguity of which "this" or assumption of which you're referring to
```suggestion
Since each replicated (and attested) MMR log is as reliable as the original, <this> can also significantly improve scalability.
```
